src/App.vue

```diff
@@ -225,7 +225,7 @@ onMounted(() => {
   <div class="flex flex-col h-full">
     <div
       v-show="expandWallet || app.expandRestoreScreen"
-      class="flex flex-col h-full bg-white-300 dark:bg-black-300 overflow-hidden"
+      class="flex flex-col h-full bg-[#EFEFEF] dark:bg-black-300 overflow-hidden"
```

Comment:
Please add this to `tailwind-config`

Reply:
[added](https://github.com/arcana-network/wallet-ui/pull/693/commits/6b3ff70f5914dc34065be1596814c2017a9c5507)